core/tests/protractor_desktop/topicAndStoryEditor.js

```diff
@@ -302,6 +302,7 @@ describe('Chapter editor functionality', function() {
 
   it('should create a basic chapter.', function() {
     storyEditorPage.createInitialChapter('Chapter 1');
+    storyEditorPage.changeNodeDescription('Chapter description 1');
```

Comment:
Can you also check that the description is visible after you reload the page? This way we can make sure the change is actually applied and persists.

Reply:
@U8NWXD the chapter description does not persists if the chapter isn't saved. This is the same for all the other fields.

Reply:
@rohitkatlaa Can we save the chapter then, either in this test or in a new one? Right now it looks like we never test that the chapter description can be changed such that the change persists. Since this is something a user would expect to be able to do, I feel like we should probably test it. Do you agree?

Reply:
I agree. So I have added a check after the save. Done.

Reply:
Thanks! this looks great